Core/MIPS/IR/IRInterpreter.cpp

```diff
@@ -74,167 +74,167 @@ u32 RunMemCheck(u32 pc, u32 addr) {
 }
 
 // We cannot use NEON on ARM32 here until we make it a hard dependency. We can, however, on ARM64.
-u32 IRInterpret(MIPSState *mips, const IRInst *inst, int count) {
+u32 IRInterpret(MIPSState *ms, const IRInst *inst, int count) {
```

Comment:
is it really necessary to rename mips to ms here and in similar places? Lots of "blame history" pollution and I don't see much benefit

Reply:
oh wait, I understand that mips is defined. But can't we just #undef it instead?

Reply:
Well, I was worried about `mips` getting depended upon elsewhere, but I think it's pretty safe within this file to undef - I went with the redefine as `mips` approach.

Still kept the changes in other files, but they're mostly just constructors anyway.

-[Unknown]